### DISCUSSION TITLE
Cabin (first map) by Ralz

### DISCUSSION THREAD

**Discussion by KennethFay** [nzp-team/nzportable#902] [Map Releases]
### [DOWNLOAD](https://archive.org/download/cabin_20240919/Cabin.zip)

My First NZP map!!
 So proud to post this online lmao

![screenfour](https://github.com/user-attachments/assets/8b1f30a5-51a1-46d8-aa58-2850f2af4964)
![screenone](https://github.com/user-attachments/assets/8de1a691-f137-424d-a810-95f76f5dc9cb)
![screenthree](https://github.com/user-attachments/assets/24e506a0-2e8f-4eed-97e5-30b7ee7c0a77)

I decided to go after a darker look when posting this map, originally it was way brigher lmaoo
 Anyways, i hope you have fun with it!!!
 
 Edit: corrected an issue related to the file not being a ".zip" but rather a ".rar" lmao

**Comment by ToonLink0001** [comment]
any ee songs?

**Comment by MotoLegacy** [comment]
please fix the scale

**Comment by MotoLegacy** [comment]
Tagging as Meets Install Standard!

**Comment by owenjames2023** [comment]
Hey this map crashes on psp when you get past wave 6

**Comment by Lonox93** [comment]
very good map
It might even be my favorite...
but I was a little surprised that the zombies spawned directly in the cabin
🫡

**Comment by Jeyphr** [comment]
Watch my playthrough here: https://youtu.be/0QSgPrFf0FM

**Comment by Boiyuh00** [comment]
this map is so fucking amazing you should totally add more perks. but the invisible wall ending early kinda gets me stuck sometimes.

**Comment by SgtCookieBobble** [comment]
Such a simple but fun map to play on the Vita; reminds me of Rust custom map on Black Ops 3 - wouldn't be saying it if I didn't believe it, but this is a first map to be proud of!
